apps/email/js/cards/compose.js

```diff
@@ -405,7 +407,7 @@ return [
       // guy in which case we really want to provide the option to delete the
       // draft.
       return (this.subjectNode.value || hasNewContent ||
-          !checkAddressEmpty() || this.composer.attachments.length ||
+          self.haveAddressesChanged() || this.composer.attachments.length ||
```

Comment:
Now that there is no inner function in this method, we can remove the var self = this, and just use `this` instead of `self` here.